test/unit/points/test-calculate-available-points.js

```diff
@@ -0,0 +1,83 @@
+const expect = require('chai').expect
+
+const DefaultContractedHours = require('../../../app/points/domain/default-contracted-hours')
+const calculateAvailablePoints = require('../../../app/points/calculate-available-points')
+const OM_TYPE_IDS = require('../../../app/points/constants/offender-manager-type-ids')
+
+describe('points/calculate-available-points', function () {
+  var nominalTarget
+  var contractedHoursPerWeek
+  var hoursReduction
+  var defaultContractedHours
+
+  beforeEach(function () {
+    nominalTarget = 7
+    contractedHoursPerWeek = 40
+    hoursReduction = 0
+    defaultContractedHours = new DefaultContractedHours(12, 7)
+  })
+
+  it('returns default PSO contracted hours when offender manager type ID is 2', function () {
+    expect(
+          calculateAvailablePoints(
+            nominalTarget, OM_TYPE_IDS.PSO, contractedHoursPerWeek,
+            hoursReduction, defaultContractedHours
+          )
+        ).to.equal(23)
```

Comment:
I would store `23` in a const so if that value ever changes down the line it only has to be updated once.